concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/Baker.java

```diff
@@ -1,32 +1,65 @@
 package com.concordium.sdk.responses.accountinfo;
 
+import com.concordium.sdk.responses.AccountIndex;
+import com.concordium.sdk.transactions.CCDAmount;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.SneakyThrows;
 import lombok.ToString;
+import org.apache.commons.codec.binary.Hex;
 
 @Getter
 @ToString
+@EqualsAndHashCode
 public final class Baker {
-    private final int bakerId;
-    private final String stakedAmount;
+    /**
+     * The baker id.
+     * Note. The baker id is non-negative.
```

Comment:
Is this comment still needed now that you have the AccountIndex type, which is by construction non-negative.

Reply:
Nice catch :-)